Roys/app/src/main/java/com/dwg_karrier/roys/Authentication.java

```diff
@@ -65,14 +66,13 @@ static String convertStreamToString(InputStream is) {
   }
 
   void authenticationAndBringPages() {
-    Dialog authDialog = new Dialog(mainContext);
+    authDialog = new Dialog(mainContext);
```

Comment:
If authDialog is only declared and used on this method, why does its scope need to place as a member variable?